### ISSUE TITLE
Sync Orders from ERPNext to Shopify

### ISSUE THREAD

**Issue by shashankshirke** [frappe/ecommerce_integrations#197]
Hi guys,

Shopify is a great shopping cart platform for **B2C sales (prepaid)** but it is also used for **B2B sales (postpaid)**. The way B2B is being achieved is by creating manual orders from the Shopify backend and then fulfilling it directly on Shopify.

This works well when Shopify is the only sales channel but when there are multiple sales channels, B2B orders are usually created in ERPNext and also fulfilled there.

Now with current integration mechanism with ERPNext, it only down-syncs the orders from Shopify to ERPNext where there doesn't seem to be a way to **up-sync orders created in ERPNext to Shopify**.

My current business case requires me to create B2B orders in ERPNext, the sync them up to Shopify where those orders can be fulfilled and then the order status is synced down to ERPNext. The reason for fulfillment on Shopify is that we get good shipping rates when we fulfill on Shopify.

So it would be great if such a workflow could be made possible in this integration.

**TL;DR version (workflow required):**
1. Create sales order in ERPNext
2. Sync the newly created order to Shopify (status: payment pending, unfulfilled)
3. Fulfill the order on Shopify (partial or full depending on stock availability)
4. Sync order status on ERPNext (and create delivery notes, sales invoices accordingly)
